app/assets/v2/js/pages/dashboard.js

```diff
@@ -280,14 +283,16 @@ var trigger_scroll = debounce(function() {
   var does_need_to_paint_more = !document.is_painting_now && !have_painted_all_bounties && ((last_active_bounty.offset().top) < (scrollPos + buffer + window_height));
 
   if (does_need_to_paint_more) {
-    paint_bounties_in_viewport(document.last_bounty_rendered + 1, document.last_bounty_rendered + 6);
+    console.log('HELLO');
```

Comment:
`console.log` to be removed
(I know you'll remove this but just giving a heads up :P )